### PR TITLE
improved and added steps for restore hub

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-12627

### CODE REVIEW

File: business_continuity/backup_restore/backup_install.adoc [hunk 1]
```diff
@@ -29,8 +29,10 @@ To enable backup operator for your active and passive hub clusters, complete the
...
 . Manually install the cluster backup and restore operator. Enable the cluster backup and restore operator (`cluster-backup`). Edit the `MultiClusterHub` resource by setting the `cluster-backup` parameter to `true`. If this hub cluster does not have the AWS Security Token Service (STS) option enabled, the OADP operator is also installed in the same namespace with the backup component. For hub clusters where the STS option is enabled, you must manually install the OADP operator. 
-. Before you run the restore operation on the passive hub cluster, complete the following steps:
-.. Ensure that the passive hub cluster uses the same {product-title-short} version or newer that the backup hub cluster uses. 
+. The restore hub cluster must use the same {product-title-short} version that the backup hub cluster uses. You cannot restore a backup on a hub cluster with a {product-title-short} version older than the one used by the backup hub cluster. If you want to run the restore operation on a hub cluster that has a {product-title-short} version newer than the cluster hub where the backup was created, complete the following:
```
Comment (jc-berger) [inline business_continuity/backup_restore/backup_install.adoc]: Given how many sub steps there are here, perhaps we should have a designated section for the restore hub and backup hub cluster versions. Something like: 

= Ensuring  restore and backup hub clusters use the same version 

The restore hub cluster must use the same {product-title-short} version that the backup hub cluster uses. You cannot restore a backup on a hub cluster with a {product-title-short} version older than the one used by the backup hub cluster. If you want to run the restore operation on a hub cluster that has a {product-title-short} version newer than the cluster hub where the backup was created, complete the following: 

1. On the restore hub....
2. Continue with all the sub steps. 

Thoughts?

Comment (swopebe) [reply business_continuity/backup_restore/backup_install.adoc]: Substeps are OK only if the higher step has a #2 -- similar to bullets. We want a peer to match it.

Comment (swopebe) [reply business_continuity/backup_restore/backup_install.adoc]: Goodness this content is long. This goes back to the "walls of text" that is in the content feedback I showed you guys. I see an entire procedure in this:

`Manually install the cluster backup and restore operator. Enable the cluster backup and restore operator (`cluster-backup`). Edit the `MultiClusterHub` resource by setting the `cluster-backup` parameter to `true`. If this hub cluster does not have the AWS Security Token Service (STS) option enabled, the OADP operator is also installed in the same namespace with the backup component. For hub clusters where the STS option is enabled, you must manually install the OADP operator.`

**Review by birsanv — APPROVED** [review]
lgtm

File: business_continuity/backup_restore/backup_install.adoc [hunk 2]
```diff
@@ -29,8 +29,10 @@ To enable backup operator for your active and passive hub clusters, complete the
...
+.. On the restore hub, install the {product-title-short} operator at the same version as the {product-title-short} hub where the backup was created.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_install.adoc]: we don't say hub, but rather hub cluster. Hub can mean multiple things when translated. The word _hub_ is the modifier of the noun, _cluster_.

Comment (swopebe) [reply business_continuity/backup_restore/backup_install.adoc]: this is throughout this set of steps.

File: business_continuity/backup_restore/backup_install.adoc [hunk 3]
```diff
@@ -29,8 +29,10 @@ To enable backup operator for your active and passive hub clusters, complete the
...
-. Before you run the restore operation on the passive hub cluster, complete the following steps:
-.. Ensure that the passive hub cluster uses the same {product-title-short} version or newer that the backup hub cluster uses. 
+. The restore hub cluster must use the same {product-title-short} version that the backup hub cluster uses. You cannot restore a backup on a hub cluster with a {product-title-short} version older than the one used by the backup hub cluster. If you want to run the restore operation on a hub cluster that has a {product-title-short} version newer than the cluster hub where the backup was created, complete the following:
+.. On the restore hub, install the {product-title-short} operator at the same version as the {product-title-short} hub where the backup was created. 
+.. Restore the date on the restore hub. 
+.. On the restore hub, use the {product-title-short} upgrade operation to upgrade {product-title-short} to the version you want to use.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_install.adoc]: you only need to use the product name here once...they are already in the product upgrade procedure and in the product doc. This line can be more concise.

File: business_continuity/backup_restore/backup_install.adoc [hunk 4]
```diff
@@ -29,8 +29,10 @@ To enable backup operator for your active and passive hub clusters, complete the
...
-. Before you run the restore operation on the passive hub cluster, complete the following steps:
-.. Ensure that the passive hub cluster uses the same {product-title-short} version or newer that the backup hub cluster uses. 
+. The restore hub cluster must use the same {product-title-short} version that the backup hub cluster uses. You cannot restore a backup on a hub cluster with a {product-title-short} version older than the one used by the backup hub cluster. If you want to run the restore operation on a hub cluster that has a {product-title-short} version newer than the cluster hub where the backup was created, complete the following:
+.. On the restore hub, install the {product-title-short} operator at the same version as the {product-title-short} hub where the backup was created. 
+.. Restore the date on the restore hub. 
+.. On the restore hub, use the {product-title-short} upgrade operation to upgrade {product-title-short} to the version you want to use. 
 .. Manually configure the hub cluster and install all operators on the active hub cluster and in the same namespace as the active hub cluster.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_install.adoc]: line 36 can be reduced and condensed, as well

File: business_continuity/backup_restore/backup_install.adoc [hunk 5]
```diff
@@ -29,8 +29,10 @@ To enable backup operator for your active and passive hub clusters, complete the
...
+. The restore hub cluster must use the same {product-title-short} version that the backup hub cluster uses. You cannot restore a backup on a hub cluster with a {product-title-short} version older than the one used by the backup hub cluster. If you want to run the restore operation on a hub cluster that has a {product-title-short} version newer than the cluster hub where the backup was created, complete the following:
+.. On the restore hub, install the {product-title-short} operator at the same version as the {product-title-short} hub where the backup was created. 
+.. Restore the date on the restore hub. 
+.. On the restore hub, use the {product-title-short} upgrade operation to upgrade {product-title-short} to the version you want to use. 
 .. Manually configure the hub cluster and install all operators on the active hub cluster and in the same namespace as the active hub cluster. 
 .. Verify that other operators, like: {aap-short}, {ocp} GitOps, or certificate manager, are installed. By verifying, you ensure that the new hub cluster is configured the same way as the initial hub cluster.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_install.adoc]: line 37 remove colon, don't use "like", punctuate with the comma and use "such as," and please use the short form of the conref or this line becomes much longer than needed. We need brevity all throughout this file but we can start with these steps.

For instance, you can remove: _By verifying, you_

File: business_continuity/backup_restore/backup_install.adoc [hunk 6]
```diff
@@ -29,8 +29,10 @@ To enable backup operator for your active and passive hub clusters, complete the
...
 .. Ensure that the passive hub cluster uses the same namespace name as the initial hub cluster when you install the backup and restore operator and any operators that are configured on the previous hub cluster.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_install.adoc]: you can just say: uses the same namespace, this line could also use some condensing. It's a bit hard to follow.

**Review by swopebe — CHANGES_REQUESTED** [review]
Mostly we need brevity on this procedure, and there are some small corrections I have requested.

File: business_continuity/backup_restore/backup_install.adoc (hunk 2 above)
Comment (swopebe) [reply business_continuity/backup_restore/backup_install.adoc]: oh, we don't say "older" either -- I raised this a while a go. Remember? Negative connotations about versioning.

Earlier, previous, later,

**Review by birsanv — APPROVED** [review]
lgtm

**Review by oafischer — APPROVED** [review]
